student.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
+
+class Student < Person
+  attr_reader :classroom
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] Kindly make sure to add `classroom` in the initialize constructor and remove the attr_reader as it's not needed.

Reply:
Made changes as suggested